Include/AppListInterface.h

```diff
@@ -9,16 +9,25 @@
 #include "UIInterface.h"
 #include "../hmi_sdk/HMI_SDK/Connect/IMessageInterface.h"
 
+struct DeviceData{
+    std::string name;
+    std::string id;
+};
+
 class AppListInterface
 {
 public:
     virtual void OnAppActivated(int appID) = 0;
     virtual void OnAppExit() = 0;
     virtual void ShowPreviousUI() = 0;
+    virtual void OnShowDeviceList() = 0;
     virtual AppDataInterface* getActiveApp() = 0;
     virtual void setUIManager(UIInterface *pUI) = 0;
     virtual void getAppList(std::vector<int>& vAppIDs, std::vector<std::string>& vAppNames) = 0;
     virtual void getAppList(std::vector<int>& vAppIDs, std::vector<std::string>& vAppNames,std::vector<std::string> &vIconPath) = 0;
```

Comment:
The most amount of characters in one line should not bigger than 80, indent in second line from one sentence should be 4 characters.